java/client/src/main/java/glide/api/commands/HashCommands.java

```diff
@@ -45,4 +45,40 @@ public interface HashCommands {
      *     If <code>key</code> does not exist, it is treated as an empty hash and it returns 0.<br>
      */
     CompletableFuture<Long> hdel(String key, String[] fields);
+
+    /**
+     * Returns the values associated with the specified fields in the hash stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/hmget/">redis.io</a> for details.
+     * @param key The key of the hash.
+     * @param fields The fields in the hash stored at <code>key</code> to retrieve from the database.
+     * @return An array of values associated with the given fields, in the same order as they are
+     *     requested.<br>
+     *     For every field that does not exist in the hash, a null value is returned.<br>
+     *     If <code>key</code> does not exist, it is treated as an empty hash, and it returns an array
+     *     of null values.<br>
+     */
+    CompletableFuture<String[]> hmget(String key, String[] fields);
+
+    /**
+     * Returns if <code>field</code> is an existing field in the hash stored at <code>key</code>.
```

Comment:
Seems like the node client has some issues in the javadoc.  I'd suggest updating this to match the python client: 
```
        Check if a field exists in the hash stored at `key`.
        See https://redis.io/commands/hexists/ for more details.

        Args:
            key (str): The key of the hash.
            field (str): The field to check in the hash stored at `key`.

        Returns:
            bool: Returns 'True' if the hash contains the specified field. If the hash does not contain the field,
                or if the key does not exist, it returns 'False'.

        Examples:
            >>> await client.hexists("my_hash", "field1")
                True
            >>> await client.hexists("my_hash", "nonexistent_field")
                False
```